README.md

```diff
@@ -93,14 +120,22 @@ The following source code exposes the `bar` function to the public API for testi
 
 
 #### Specifying different start and end comment values
-The following configuration will strip out code that begins with the `/* start-test-block */` comment and ends with the `/* end-test-block */` comment from all `.js` files in the `dist/` folder.
+The following configuration will strip out code that begins with the `/* start-test-block */` comment and ends with the `/* end-test-block */` comment, and code that begins with the `<!-- start-html-test-code -->` comment and ends with the `<!-- end-html-test-code -->` comment from all `.js` files in the `dist/` folder.
 
 ```js
 grunt.initConfig({
   strip_code: {
     options: {
-      start_comment: 'start-test-block',
-      end_comment: 'end-test-block',
+      blocks: [
+        {
+          start_block: "/* start-test-block */",
+          end_block: "/* end-test-block */"
+        },
+        {
+          start_block: "<!-- start-html-test-code -->",
+          end_block: "<!-- end-html-test-code -->"
+        }
+      ]
     },
     src: 'dist/*.js'
```

Comment:
If you need to strip most of your files but not all of them just create separate targets inside strip_code task. Also you can filter files that will be processed by `filter` prop (http://gruntjs.com/configuring-tasks#files).

This plugin does 1 job --stipping. If you have difficulties with separating which files should be stripped and which should not that this puzzle should be solved at different 'level'.

I hope I understand you example and hope my answer can help you..